source/content/terminus/02-install.md

```diff
@@ -53,9 +53,9 @@ Terminus does not work with the following platforms:
 
 <Alert title="PHP 8.4 and Terminus" type="danger">
 
-The current stable version of Terminus, 3.x, is not compatible with PHP 8.4. We recommend using PHP 8.3 for Terminus 3.x.
+Terminus, 3.x, is not compatible with PHP 8.4. If you are using PHP <= 8.3 you should use Terminus 3.x.
```

Comment:
This should say `PHP <= 8.1`, as Terminus 4 works with PHP 8.2+.  We should probably also recommend folks upgrade if they're in this situation.